src/ert/scheduler/job.py

```diff
@@ -176,7 +176,9 @@ async def _max_runtime_task(self) -> None:
         )
         assert self._scheduler._events is not None
         await self._scheduler._events.put(to_json(timeout_event))
-
+        logger.error(
+            f"Realization {self.iens} stopped due to MAX_RUNTIME={self.real.max_runtime} seconds"
```

Comment:
Shouldn't this be the same message as you would get when using job queue? I think it would be great to have a seamless transition between jobqueue and scheduler, including the log messages. Maybe we should update the JobQueue to match this one.

Reply:
Yes, I found the job_queue message a little unclear. Can update it.

Reply:
Yes, I agree